views/widget.scout.display.php

```diff
@@ -19,20 +19,21 @@
     // Grab the settings from $instance and fill out default values as needed.
 	$title = empty( $instance['title'] ) ? ' ' : apply_filters( 'widget_title', $instance['title'] );
 ?>
-	<script type="text/javascript">
+<?php if ($use_spm): ?>
+    <div class="sailthru-spm-widget">
+         <div id="<?php echo $this->id; ?>"></div>
+          <script type="text/javascript">
+            jQuery(function() {
+                SPM.addSection('<?php echo $section; ?>', {
```

Comment:
The widget still uses scout when the customer has Horizon. The cleanup of file names will be done in a later release when fully deprecated. 

There has to be a section present for the user to save the widget that would be rendered.